src/main/java/com/google/sps/data/Response.java

```diff
@@ -33,18 +33,24 @@ public Response(String status, List<Restaurant> results) {
         this.results = results;
     }
 
-    public String status() {
-        return status;
+    public String getStatus() {
+        return this.status;
     }
 
-    public List<Restaurant> results() {
-        return results;
+    public List<Restaurant> getResults() {
+        return this.results;
     }
 
     public void pick() {
-        int randIdx = (int) (Math.random() * results.size());
-        pick = results.get(randIdx);
-        results.remove(randIdx);
+        if(results.size() > 1){
+            int randIdx = new Random().nextInt(results.size()-1);
```

Comment:
you can remove the clunky if else if you use a java one line if-else statement :)

Reply:
fixed